connector/github/github.go

```diff
@@ -34,16 +34,22 @@ type Config struct {
 	ClientID     string `json:"clientID"`
 	ClientSecret string `json:"clientSecret"`
 	RedirectURI  string `json:"redirectURI"`
-	Org          string `json:"org"`
+	Orgs         []Org  `json:"orgs"`
 	HostName     string `json:"hostName"`
 	RootCA       string `json:"rootCA"`
 }
 
+// Org holds org-team filters, in which teams are optional
```

Comment:
Need to explain the impact of providing the teams field. Also the logic about what teams or orgs a user has to belong to to login.